tests/testthat/test_surv_measures.R

```diff
@@ -0,0 +1,43 @@
+context("survival measures")
+
```

Comment:
I prefer splitting into more files as well. We're already running into issues with tests being too large.